exonum-java-binding/core/src/main/java/com/exonum/binding/runtime/FrameworkModule.java

```diff
@@ -33,7 +32,6 @@ protected void configure() {
     bind(Server.class).toProvider(Server::create).in(Singleton.class);
     bind(ViewFactory.class).toInstance(ViewProxyFactory.getInstance());
     // todo: Consider providing an implementation of a Node —
-    // requires changing its contract.
-    bind(UserServiceAdapter.class);
+    //   requires changing its contract.
```

Comment:
nit: for some reason you add a space here. Is this comment needed at all?

Reply:
To make it a [multiline todo](https://www.jetbrains.com/help/idea/using-todo.html).

I'd keep it till dynamic services kick in.